core/loadpoint.go

```diff
@@ -651,64 +651,69 @@ func (lp *Loadpoint) Prepare(uiChan chan<- util.Param, pushChan chan<- push.Even
 	}
 }
 
+func (lp *Loadpoint) setAndPublishEnabled(enabled bool) {
+	if enabled != lp.enabled {
+		lp.log.DEBUG.Printf("charger %s", status[enabled])
+		lp.enabled = enabled
+	}
+	lp.publish(keys.Enabled, enabled)
+}
+
 // syncCharger updates charger status and synchronizes it with expectations
 func (lp *Loadpoint) syncCharger() error {
 	enabled, err := lp.charger.Enabled()
 	if err != nil {
 		return err
 	}
 
-	// some chargers (i.E. Easee in some configurations) disable themself to be able to switch phases
-	if !enabled && lp.enabled && !lp.phaseSwitchCompleted() {
-		return lp.charger.Enable(true) // enable charger
-	}
+	consistentState := lp.chargerUpdateCompleted() && lp.phaseSwitchCompleted()
 
-	if lp.chargerUpdateCompleted() {
+	if consistentState {
 		defer func() {
-			lp.enabled = enabled
-			lp.publish(keys.Enabled, lp.enabled)
+			lp.setAndPublishEnabled(enabled)
 		}()
 	}
 
+	// #1: check charger logic, fix charger state if necessary (for chargers that start charging while being disabled)
```

Comment:
Warum steht dieser IF-Block alleine, während die unten eigentlich ein case Statement sind?

Reply:
Weil dieser Block nur dazu dient den State in der Charger Instanz mit der Physik zu synchronisieren. Es hat aber keine „schaltende“ Wirkung. Daher kann dieser geschehen wenn nötig.